src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -243,6 +243,23 @@ public string StyleName
             }
         }
 
+        /// <summary>
+        /// The KeyInputFocus, type bool.
```

Comment:
hmm.. `KeyInputFoucs` type is string...

Reply:
Changed to 'bool' type of `KeyInputFocus`